frontend/src/app/shared/components/sidemenu/sidemenu.component.html

```diff
@@ -23,34 +23,52 @@
   [ngClass]="{'op-sidemenu--items_collapsed' : collapsed}"
 >
   <li
+    *ngFor="let item of items"
     class="op-sidemenu--item"
     data-qa-selector="op-sidemenu--item"
-    *ngFor="let item of items"
   >
+    <op-sidemenu
+      *ngIf="item.children && item.children.length"
+      [title]="item.title"
+      [items]="item.children"
+      [collapsible]="item.collapsible"
+    ></op-sidemenu>
+
     <a
+      *ngIf="!item.children && item.uiSref"
       class="op-sidemenu--item-action"
+      [class]="{ 'op-sidemenu--item-action_without_icon': !item.icon }"
```

Comment:
apart from this small change, the changes in here are done in commit `95679d470e6c0d171e025c25a99ab7eea2e29c9c` and were cherry picked to this branch to get it working.

Reply:
I think no-icon should be the default. In any case, it should be possible to do what you want in css without having to set the class, I think

Reply:
I'd argue against that. A class is always a better option than pseudo selectors imho.

Reply:
I think ease of use trumps that. This pseudo selector adds one non-class item of specificity, but makes sure the html structure is more self-explanatory. I'm okay with going for a class, but in that case I'd argue it should be `_with-icon`

Reply:
Alright, I could flip the logic, so that paddings are by default considering no icons.